src/services/dashboards/default-dashboard/helper.ts

```diff
@@ -0,0 +1,21 @@
+import { ERROR_CASE_WIDGET_INFO } from '@/services/dashboards/default-dashboard/config';
+import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/config';
+import { getWidgetConfig } from '@/services/dashboards/widgets/widget-helper';
+
+export const getDashboardLayoutWidgetInfoList = (widgetList: string[]): DashboardLayoutWidgetInfo[] => widgetList.map((widgetId) => {
+    try {
+        const widgetConfig = getWidgetConfig(widgetId);
+        const widgetInfo: DashboardLayoutWidgetInfo = {
+            widget_name: widgetConfig.widget_config_id,
+            title: widgetConfig.title ?? widgetConfig.widget_config_id,
+            widget_options: widgetConfig.widget_options ?? {},
+            size: widgetConfig.sizes[0],
+            version: '1',
```

Comment:
question: Do we have to specify `version` when request real API?

Reply:
Hm.. I don't think so.. I think we can omit version in each widget info, and just specify to dashboard info. What do you think about it?